hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1098,6 +1098,16 @@ public String getClientRpcPort() {
     return addr == null ? "0" : Integer.toString(addr.getPort());
   }
 
+  public String getBlockProtocolRpcPort() {
+    InetSocketAddress addr = getBlockProtocolServer().getBlockRpcAddress();
+    return addr == null ? "0" : Integer.toString(addr.getPort());
```

Comment:
Do we need these nulll check?

Reply:
Following existing norm here followed for client port :
 @Override
  public String getClientRpcPort() {
    InetSocketAddress addr = getClientRpcAddress();
    return addr == null ? "0" : Integer.toString(addr.getPort());
  }